backend/src/Models/ElectionRolls.ts

```diff
@@ -44,14 +45,15 @@ class ElectionRollDB {
     }
 
 
-    submitElectionRoll(election_id: number, voter_ids: string[], submitted: Boolean, state: string, history: ElectionRollAction): Promise<boolean> {
+    submitElectionRoll(election_id: number, voter_ids: string[], submitted: Boolean, state: string, history: ElectionRollAction, registration: any): Promise<boolean> {
```

Comment:
I'm assuming the option to pass multiple voter_ids is ONLY used when setting up the election?

If so would consider breaking that into a separate method on the interface.

Reply:
Yea we really only do multiple when they're added in batch by the election host when setting up. Adding a separate method would be better for adding just one. Or I can refactor to input an array of the election rolls and move the mapping of voter ids to the controller.

Reply:
Updated to just pass in list of voter rolls.